pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -331,6 +331,10 @@ self: super: ({
   # Tests fail on macOS https://github.com/mrkkrp/zip/issues/112
   zip = dontCheck super.zip;
 
+  snap = super.snap.overrideAttrs (drv: {
```

Comment:
That's snap, not warp?

Reply:
Derp, botched the commit message. 
Was thinking of warp because I ended up re-testing the sandbox vs local suite interaction with it as well.
Fixed